api/workspace/v1alpha1/workspace_types.go

```diff
@@ -65,7 +65,7 @@ type Config struct {
 
 // NetworkRule is an abstract network configuration rule for workspace
 type NetworkRule struct {
-	PortName         string  `json:"portName"`
+	NetworkRuleName  string  `json:"networkRuleName"`
```

Comment:
In `NetworkRule` struct, just `Name` is simple and makes sense

Reply:
fixed😃